config/test/affine.yaml

```diff
@@ -18,6 +18,7 @@ train:
   preprocess:
     batch_size: 2
     shuffle_buffer_num_batch: 1
+    num_parallel_calls: -1 # number elements to process asynchronously in parallel during preprocessing, -1 means unlimited, heuristically it should be set to the number of CPU cores available
```

Comment:
Why should this be the number of available cores?

Reply:
TensorFlow said so. will update in the docstring